tornado-unittests/src/main/java/uk/ac/manchester/tornado/unittests/api/TestSharedBuffers.java

```diff
@@ -335,38 +378,62 @@ public void testThreeTaskGraphsWithSharedBuffers() throws TornadoExecutionPlanEx
         }
     }
 
-    /**
-     * Tests the persistence of buffers across multiple sequential task graphs with context state management.
-     *
-     * <p>This test demonstrates a four-stage processing pipeline where both intermediate result data and
-     * context state are maintained across task graph executions. It showcases:</p>
-     *
-     * <ul>
-     * <li>Data flow between consecutive task graphs using persistent buffers</li>
-     * <li>Context initialization, updating, and finalization across multiple graphs</li>
-     * <li>Proper pipeline execution with sequential dependencies</li>
-     * <li>The required "touch" operation before transferring results back to host</li>
-     * </ul>
-     *
-     * <p><strong>API Requirements:</strong></p>
-     * <ul>
-     * <li>Buffers must be explicitly marked with {@code persistOnDevice()} to remain accessible</li>
-     * <li>Consuming graphs must use {@code consumeFromDevice()} with the source graph's name</li>
-     * <li>All buffers being consumed must be listed explicitly in the consume call</li>
-     * <li>Graphs must be executed in dependency order</li>
-     * <li>A "touch" operation is required on output buffers before transfer to ensure proper copy-back</li>
-     * </ul>
-     *
-     * <p><strong>Limitations:</strong></p>
-     * <ul>
-     * <li>No automatic scheduling - execution order must match data dependencies</li>
-     * <li>No implicit persistence - all buffers to persist must be explicitly listed</li>
-     * <li>Graph names must be consistent between producer and consumer</li>
```

Comment:
In this case, we don;t need to specify the graph names to create a dependency between a consumer and a persistent graph.